Makefile

```diff
@@ -44,7 +44,7 @@ SETUP_ENVTEST := go run sigs.k8s.io/controller-runtime/tools/setup-envtest
 K8S_ENVTEST_VERSION := 1.21.4
 
 PACKAGE=github.com/openshift/external-dns-operator
-MAIN_PACKAGE=$(PACKAGE)/cmd/external-dns-operator
+MAIN_PACKAGE=$(PACKAGE)
```

Comment:
`$(PACKAGE)` and `$(MAIN_PACKAGE)` has same value. Is there a reason to keep both of them?

Reply:
Not anymore, removed `MAIN_PACKAGE`

Reply:
What was the reason for moving the main executable out of `cmd/`?  Is this change directly related somehow to the v1beta1 migration?